### PR TITLE
copied code and installed linters

### PR DESCRIPTION
**Pull request by ishpaul777**
In this pull request I copied code from trials and installed linters

### CODE REVIEW

**Review by ishpaul777 — COMMENTED** [review]
there are linter errors in the styles.css and index.html file

File: styles.css [hunk 1]
```diff
@@ -1 +1,163 @@
...
+#header
+{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 1.5rem;
+    /*background: #009688;*/
+    height: 80px;
+}
+#nav-bar
```
Comment (ishpaul777) [inline styles.css:30]: there is empty blocks of selector which can be avoided

File: styles.css [hunk 2]
```diff
@@ -1 +1,163 @@
...
+}
+.below-heading 
+{
+    margin: 20px auto;
+}
+
+.inputs
+{
+    d
+}
```
Comment (ishpaul777) [inline styles.css:58]: empty blocks style selector

**Review by ishpaul777 — COMMENTED** [review]
the web page design is not looking as it should be according to the given webpage

**Review by ishpaul777 — COMMENTED** [review]
the right git flow is not used

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight
- [x] You've done a great job to address the issues a student needs to fix in a polite way and provided a reason it is important to fix them.

## Required Changes ♻️
- [x] You should review your project documentation for it is missing some important part as the description, and explain to a student to make those changes.

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @ishpaul777  👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🌤️ 



 ## Highlights

✔️ Nice and positive reviews

 ## Required Changes ♻️

- [x] What the previous reviewer meant for you to do was to give a review on the documentation, not add  the documentation by yourself. You just had to point out the documentation is not there and why adding documentation is good practice. __Remember to always give a why behind all your changes__ and kindly make a review on the documentation now.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishpaul777 — COMMENTED** [review]
Proper Readme.md documentation not added by the student which is a good practice helps other people to know about the project if they are Viewing or reviewing their project

**Review by zhadier — APPROVED** [review]
## Hi @ishpaul777  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional
<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**